bookkeeper-benchmark/src/main/java/org/apache/bookkeeper/benchmark/package-info.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+/**
+ * @TODO: Write JavaDoc comment
```

Comment:
@eolivelli, I agree with you about the TODOs can be harmful, but they are not, when the team follows the [boy scout rule](http://programmer.97things.oreilly.com/wiki/index.php/The_Boy_Scout_Rule).
If you like clean code and want it, you need to do for it.
TODO comment is a code smell, so if you can clean it up, lets do it.
It is a tool to feel worse. It drives you to do the cleaning.
Eg.: If you don't have anything like checkstyle or pmd or something like that, your code base will have a lot of issues like these. After it is enabled you need to keep it clean. TODOs are the same, I think.
"If you complain, but you do not do anything, it does not hurt enough." - Dr. Bubó